### PR TITLE
Implement warning and error query

### PR DESCRIPTION
**Pull request by GiasemiSh**
Issue: https://github.com/QMSTR/qmstr-all/issues/101

### CODE REVIEW

File: pkg/analysis/analyzer.go [hunk 1]
```diff
@@ -82,3 +82,17 @@ func (a *Analyzer) RunAnalyzerModule() error {
 	}
 	return nil
 }
+
```
Comment (hemarkus) [inline pkg/analysis/analyzer.go]: Where is the structure the commit message mentions?

File: pkg/database/dgraph.go [hunk 2]
```diff
@@ -147,24 +147,6 @@ func queueWorker(db *DataBase) {
 	}
 }
 
-func (db *DataBase) AlterFileNode(node *service.FileNode) (string, error) {
```
Comment (hemarkus) [inline pkg/database/dgraph.go]: Why are those functions deleted? Why there is nothing in the commit message?

File: pkg/service/common.go [hunk 3]
```diff
@@ -48,8 +48,8 @@ func SanitizeAnalyzerNode(analyzerNode *Analyzer) {
 	analyzerNode.NodeType = NodeTypeAnalyzerNode
 }
 
-func SanitizePackageNode(pkgNode *PackageNode, dbpkgNode *PackageNode) {
-	pkgNode.Uid = dbpkgNode.Uid
-	pkgNode.Session = dbpkgNode.Session
-	pkgNode.NodeType = NodeTypePackageNode
+func SanitizePackageNode(pkgNode *PackageNode) {
+	for _, info := range pkgNode.AdditionalInfo {
```
Comment (hemarkus) [inline pkg/service/common.go]: This is not equivalent. Why did you deleted the checks for session, nodetype etc?

Comment (GiasemiSh) [reply pkg/service/common.go]: Have a look now at the commit message.

**Comment by GiasemiSh** [comment]
@hemarkus Please review again

**Comment by hemarkus** [comment]
retest this please

**Pull request closed without merging**